workflow/snakemake_rules/main_workflow.smk

```diff
@@ -1360,13 +1360,27 @@ rule build_description:
                 > {output.description:q}
         """
 
+def get_auspice_config(w):
+    """
+    Auspice configs are chosen via this heirarchy:
+    1. A build-specific JSON
+    2. A `make_auspice_config` rule, if it exists
+    3. The file specified in the `config.files` dict
+    """
+    if "auspice_config" in config["builds"].get(w.build_name, {}):
+        return config["builds"][w.build_name]["auspice_config"]
+    if "make_auspice_config" in rules.__dict__:
+        return rules.make_auspice_config.output
```

Comment:
Would probably call this rule just `auspice_config` to align with how we name ~all other rules (all of which "make" something :-).

Also, `rules.__dict__` is more idiomatically spelled `vars(rules)` FWIW, but personally I don't think it much matters either way.

Reply:
Sure! Done.